### ISSUE TITLE
Add Swift styleguide.

### ISSUE THREAD

**Issue by jiashuw** [envoy/styleguides#21]

**Comment by fangpenlin** [comment]
As we are pretty small team, it would be hard and not efficient for us to come up with a complete styling guideline from ground up, I guess it makes more sense to look into other companies' public style guideline, pick one we all feel comfortable with, or maybe change one or two things that we don't like. Or maybe we can also pick the things we like from different style guideline and built our own.

@LFabien what do you think? do you have any preferred style guide line you used before?

**Comment by jiashuw** [comment]
@fangpenlin yeah that's what i meant, sorry for the confusion - we should just fork others' styleguide and adjust to fit our needs

**Comment by fangpenlin** [comment]
@jiashuw yeah, for sure, I guess we can starting from collecting a list of style guideline here I found some of them

https://github.com/github/swift-style-guide
https://github.com/raywenderlich/swift-style-guide
https://github.com/linkedin/swift-style-guide
https://github.com/eure/swift-style-guide
